src/emit.ts

```diff
@@ -50,30 +60,44 @@ export class Emit {
     core.setOutput('revision-id', revId);
   }
 
-  async build(data: Build): Promise<void> {
-    const job = {uid: data.uid, source: SOURCE};
-    const buildKey = {uid: data.uid, job};
+  async build(build: Build): Promise<void> {
+    const orgKey = {uid: build.org.toLowerCase(), source: BUILD_SOURCE};
+    const pipelineId = `${build.org}/${build.repo}/${build.workflowName}`.toLowerCase();
```

Comment:
@tovbinm @cjwooo This can be a way to generate the pipeline key bit haven't found a way to get the unique workflowId from the github action context. Maybe ok since it looks like the Id changes when the config changes. 

However two totally different flows can have the same name so 🤷🏽

Reply:
I dont have a better way. So the user can override it through with this - https://github.com/faros-ai/faros-cicd-github-action/pull/47/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R38 ?